synapseclient/core/utils.py

```diff
@@ -108,7 +118,8 @@ def extract_filename(content_disposition_header, default_filename=None):
     """
     Extract a filename from an HTTP content-disposition header field.
 
-    See `this memo <http://tools.ietf.org/html/rfc6266>`_ and `this package <http://pypi.python.org/pypi/rfc6266>`_
+    See `this memo <http://tools.ietf.org/html/rfc6266>`_ and
```

Comment:
These URL links will need to be updated:

[display text](link)

```suggestion
    See [this memo](http://tools.ietf.org/html/rfc6266) and
```